Kbs.Wpf/Boat/Components/BoatStatusesViewModel.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Kbs.Business.Boat;
+
+namespace Kbs.Wpf.Boat.Components;
+
+public class BoatStatusesViewModel
```

Comment:
Must inherit from ViewModel:
```suggestion
public class BoatStatusesViewModel : ViewModel
```